packages/core/android/src/main/kotlin/com/datadog/reactnative/DatadogSDKWrapper.kt

```diff
@@ -21,6 +26,45 @@ import com.datadog.android.trace.Trace
 import com.datadog.android.trace.TraceConfiguration
 import com.datadog.android.webview.WebViewTracking
 
+/**
+ * Internal object used to add internal testing.
+ */
+object DatadogSDKWrapperStorage {
+    internal val onInitializedListeners: MutableList<(InternalSdkCore?) -> Unit> = mutableListOf()
+
+    /**
+     * Adds a Listener called when the core is initialized.
+     */
+    fun addOnInitializedListener(listener: (InternalSdkCore?) -> Unit) {
+        onInitializedListeners.add(listener)
+    }
+
+    /**
+     * Exposed for testing purposes only.
+     */
+    fun notifyOnInitializedListeners() {
+        onInitializedListeners.forEach {
+            it(core)
+        }
+    }
+
+    private var core: InternalSdkCore? = null
```

Comment:
This needs to be pulled up. Class layout should have properties declaration first, then functions